test/functional/plugin/lsp/lsp_spec.lua

```diff
@@ -127,25 +131,48 @@ describe('Language Client API', function()
      -- exec_lua("lsp.stop_all_clients(true)")
     end)
 
-    describe('start_client and stop_client', function()
-      it('should return true', function()
-        for _ = 1, 20 do
-          helpers.sleep(10)
-          if exec_lua("return #lsp.get_active_clients()") > 0 then
-            break
-          end
-        end
```

Comment:
@norcalli @tjdevries @h-michael just FYI, can use `retry()` instead of this for-loop (see below)